### ISSUE TITLE
Running Jupyter at NSCC

### ISSUE THREAD

**Issue by happinesszl** [nus-mornin-lab/datathon-nscc-docker#1]
Hi, Kien. Thanks for sharing this Dockfile.
I am a PhD student at NUS Physics. I am trying to use your image at NSCC(kiend/datathon-nscc:cuda9.0-ubuntu16.04), but the method to run Jupyter at NSCC in your readme file doesn't work for me. The error message is:

    INFO: Submitting interactive job to PBS
    INFO: qsub -I -q fji -l walltime=2:00:00 -P41000001 -- /usr/local/bin/nscc-docker run -t --net=host
    qsub:	standard input and output must be a terminal for
	    interactive job submission

Then I tried:

```
qsub -I -q gpu -l walltime=2:00:00 -P Personal -- /usr/local/bin/nscc-docker run -t --net=host    kiend/datathon-nscc:cuda9.0-ubuntu16.04 << EOF
    export HOME=$HOME
    export USER=$USER
    cd $HOME
    jupyter lab --LabApp.ip='0.0.0.0'
EOF
```

The error message is:

    qsub:	standard input and output must be a terminal for
	    interactive job submission

I would appreciate any advice, thanks.

**Comment by kiendang** [comment]
To use it with an interactive job you run
```bash
qsub -I -q fji -l walltime=2:00:00 -P Personal 
```
After the command finishes you run
```bash
nvidia-docker-run --net=host kiend/datathon-nscc:cuda9.0-ubuntu16.04 << EOF
    export HOME=$HOME
    export USER=$USER
    cd $HOME
    jupyter lab --LabApp.ip='0.0.0.0'
EOF
```
then follow the instructions in NSCC's DGX guides (I assume you have this?) to configure port forwarding (It's the part about ssh) so you can access jupyter in your browser.

To submit a PBS job I used a script. I will try to find it tmr then get back to you.

**Comment by happinesszl** [comment]
Thank you for your prompt reply! It clarifies my doubt.

Sorry, I didn’t find the NSCC’s DGX guide in their website (https://help.nscc.sg/user-guide/). It would be great if you could send me a copy of that (my email is linzhang@u.nus.edu).

I would really appreciate if you could send me a sample script to submit a PBS job. It could save me a lot of time.

**Comment by kiendang** [comment]
Just sent you an email

**Comment by happinesszl** [comment]
Hi Kien,I have arrived at Waa Cow. But they closed today...
Lin
    	
    	

    	
    
  




On Fri, Nov 2, 2018 at 11:00 AM +0800, "Dang Trung Kien" <notifications@github.com> wrote:












Closed #1.

**Comment by kiendang** [comment]
Hi Lin,

Im coming to utown now. Do you have any other place in mind?
